lib/linguist/languages.yml

```diff
@@ -1941,6 +1941,16 @@ HiveQL:
   tm_scope: source.hql
   ace_mode: sql
   language_id: 931814087
+HolyC:
+  type: programming
+  color: "#FFF47C"
+  extensions:
+  - ".hc"
+  - ".hc.z"
```

Comment:
Is this a compressed version of `.hc`? If so we don't need this extension as the file will be binary so ignored anyway.